api/events/swagger-api_events.yaml

```diff
@@ -9,6 +9,20 @@ tags:
 - name: Events
   description: Events management
 paths:
+  /events/actions:
+    get:
+      tags:
+      - Actions
+      summary: Get the list of all possible actions
+      responses:
+        200:
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  type: string
```

Comment:
Is not the same as you code in the paper card?
  /cards/actions:
    get:
      tags:
      - Actions
      summary: Get the list of all possible actions
      responses:
        200:
          description: successful operation
          content:
            application/json:
              schema:
                type: array
                items:
                 ** $ref: '#/components/schemas/Actions' **

Reply:
You are right, it was a bug, it is fixed in the PR #159